documentation/commands/add-stream.md

```diff
@@ -23,7 +23,7 @@ following bits defined:
 * 0x01 (Takeover) - Specifies that the stream should send over all remaining data to the remote node and then set the remote nodes vbucket to active state and the source nodes vbucket to dead.
 * 0x02 (Disk Only) - Specifies that the stream should only send items only if they are on disk. The first item sent is specified by the start sequence number and items will be sent up to the sequence number specified by the end sequence number or the last on disk item when the stream is created.
 * 0x04 (Latest) - Specifies that the server should stream all mutations up to the current sequence number for that VBucket. The server will overwrite the value of the end sequence number field with the value of the latest sequence number.
-* 0x08 (No Value) - Specifies that the server should stream only item key and metadata in the mutations and not stream the value of the item
```

Comment:
I'd be tempted to leave this in but mark as "deprecated, removed in 5.0" or similar just to help anyone which looks at 4.5.x code.